backend/src/api/routes/authRoutes.ts

```diff
@@ -1,10 +1,12 @@
 import { Router, Request } from 'express';
 import { run } from '../../common/utils/routeHelper';
 import { register, refreshTokens, login, forgotPassword, resetPassword } from '../../services/authService';
+import { getUserById } from '../../services/userService';
 
 const router = Router();
 
 router
+  .get('/me', run((req: Request) => getUserById(req.body.id)))
```

Comment:
id should be taken from req.user, do not send it in body

Reply:
fixed